.github/workflows/release.yaml

```diff
@@ -328,11 +328,74 @@ jobs:
           event-type: coder-release
           client-payload: '{"coder_version": "${{ steps.version.outputs.version }}"}'
 
+  publish-homebrew:
+    name: Publish to Homebrew tap
+    runs-on: ubuntu-latest
+    needs: release
+    if: ${{ !inputs.dry_run }}
```

Comment:
If you want to keep this as a separate job in the workflow, you'll need to download the assets like we do for winget. You could also move the step to be in the release job above instead of in it's own job since it has the release assets.

Reply:
wrote a pretty nice script to just download the checksums.txt file and pull them from there. think it's best that these all live in their own jobs. thanks for the catch!